frontend/src/pages/dashboard/constants.ts

```diff
@@ -70,14 +72,14 @@ export interface Personnel {
   workLocation?: Location;
   homeLocation: Location;
   experiences?: ExperienceInterface[];
-  availability?: AvailabilityInterface[];
+  availability?: Availability[];
   status: Status;
   newMember?: boolean;
   willingToTravel: boolean;
   remoteOnly: boolean;
   unionMembership: string;
   ministry: string;
-  applicationDate?: Date;
+  applicationDate?: string;
```

Comment:
The Personnel.applicationDate is used with 2 functions: datePST and dayjs.format.
datePST parses the date with PST timezone but dayjs parses in the users local time zone as we have not passed a timezone there.